Dockerfile

```diff
@@ -11,5 +11,4 @@ EXPOSE 9011
 EXPOSE 9012
 EXPOSE 9013
 COPY --from=builder /go/croc/croc /croc
-ENTRYPOINT ["/croc"]
-CMD ["relay"]
+CMD ["sh", "-c", "/croc --pass $CROC_PASS relay"]
```

Comment:
this would set the password to "relay" and omit the last parameter when the user does not specify a password on his own.

Better would be a custom entrypoint script that builds up the startup argument. 

Something like:
```
#!/bin/sh
set -e
if [ -n "$CROC_PASS " ]; then
    set -- "$@" --pass "$CROC_PASS "
fi
exec croc "$@"
```

Reply:
Thanks, didn't think about that one. Should be better now.